docs/source/algorithms/LineProfile-v1.rst

```diff
@@ -24,6 +24,13 @@ By default, the profile is calculated as an average over the line width. This be
 'Sum'
     Sum the values, weighting them by :math:`n / n_{tot}` where :math:`n` is the number of summed data points (excluding special values if *IgnoreNans* or *IgnoreInfs* is set) and :math:`n_{tot}` is the total number of data points (including special values).
 
+Profiles over distributions
+---------------------------
+
+Horizontal profiles over distribution data (Y divided by bin width) will be distributions as well.
+
+However, vertical profiles over distribution data will be non-distributions, that is, the Y values will be multiplied by the X bin width before summation or averaging.
```

Comment:
I am not entirely convinced that this makes sense. Can you explain the reasoning behind this? I agree that the output would not be a distribution, but does that really imply that we need to multiply by the X bin width?

Reply:
The line profile is a single histogram, and in the case of vertical profiles, the cut is transposed. The problem with vertical line profiles from distributions is that if the bin width normalization is kept, there is no flag or similar in Mantid that could tell that the bins in the output workspace are normalized by the bin 'height'. That is why I opted to unnormalize the bins. I am also unsure if this is the right thing to do, so let me know if you better ideas.

Reply:
I am not sure either.... but the bin 'height' is not available in the output workspace, i.e., in the best case (constant input bin width) it contains an unknown normalization factor, or in the worst case (variable input bin width) each output value will contain a different unknown factor that is impossible to deduce.

If we convert to input to a distribution, at least the value make sense, just the unit is unknown?

@martyngigg Any thoughts on this?

Reply:
The 'height' is actually available, as the vertical axis is a `BinEdgeAxis`. The algorithm doesn't accept variable bin widths as input.

Reply:
Ok, with `CommonBinsValidator` this seems to make sense.

Reply:
Agreed that with common bins this makes sense as the best we can achieve with the current strategy for tracking normalization.